### DISCUSSION TITLE
How do I add a space here?

### DISCUSSION THREAD

**Discussion by vamonk** [mkdocs-publisher/mkdocs-publisher#103] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/fc879258-7810-4b69-aab9-ab9201ad2f19)

In Obsidian: 

![image](https://github.com/user-attachments/assets/361cb2b7-1b68-46ed-af78-0869d2b0e86e)

**Comment by mkusz** [comment]
It's an issue in the plugin code. I need to create a fix for that.

**Reply by vamonk** [reply]
Now I have a lot of text to write, but next week I will have to publish to clients. I hope you find the time ;-)

**Reply by vamonk** [reply]
> It's an issue in the plugin code. I need to create a fix for that.

Could I possibly help myself? I want to publish tomorrow. It doesn't look good like this.

**Comment by mkusz** [comment, marked as answer]
It's now should be fixed by #105
